### PR TITLE
Add missing README.md and LICENSE

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
files are placed at:

```
% diff -u ../before.txt ../after.txt
--- ../before.txt       2020-06-12 11:38:52.732569486 +0900
+++ ../after.txt        2020-06-12 11:59:11.748325613 +0900
@@ -7054,6 +7054,10 @@
 /usr/lib/tmpfiles.d/td-agent.conf
 /usr/sbin/td-agent
 /usr/sbin/td-agent-gem
+/usr/share/doc/td-agent
+/usr/share/doc/td-agent/README.md
+/usr/share/licenses/td-agent
+/usr/share/licenses/td-agent/LICENSE
 /usr/share/man/man1/td-agent-gem.1.gz
 /usr/share/man/man1/td-agent.1.gz
 /usr/share/man/man1/td.1.gz
```

**Comment by cosmo0920** [comment]
I'm building RPM packages.

**Comment by cosmo0920** [comment]
CentOS 6 building is failed:
https://github.com/clear-code/td-agent-builder/pull/100/checks?check_run_id=764083002#step:5:14232

**Comment by cosmo0920** [comment]
CentOS 7 and 8 package are good. :+1:

**Comment by cosmo0920** [comment]
It seems that CentOS 6 does not have `%license` directive. So, we should use `%doc` for LICENSE file installation on CentOS 6. 
https://github.com/clear-code/td-agent-builder/pull/100/files#diff-1bbb40b55cf61387ad828ebaa0b0e41eR167-R169

File: td-agent/yum/td-agent.spec.in [hunk 1]
```diff
@@ -162,7 +162,11 @@ fi
 
 %files
 %doc README.md
+%if %{?use_systemd}
```
Comment (ashie) [inline td-agent/yum/td-agent.spec.in:165]: We should use OS version or define another macro since it's not depend on systemd.

**Comment by kenhys** [comment]
I've rebased with master and push one commit.

File: td-agent/yum/td-agent.spec.in [hunk 2]
```diff
@@ -161,6 +162,12 @@ elif [ -f "%{_sysconfdir}/prelink.conf" ]; then
 fi
 
 %files
+%doc README.md
+%if %{?use_license}
```
Comment (cosmo0920) [inline td-agent/yum/td-agent.spec.in:166]: :+1:

**Pull request merged**